src/main/java/DateTimeManager.java

```diff
@@ -0,0 +1,18 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class DateTimeManager {
+    private static LocalDateTime current = LocalDateTime.now();
```

Comment:
I think perhaps currentTime or something to that effect might be better in terms of clarifying what the current variable refers to.

Reply:
Updated the naming, thanks!